Library/Homebrew/cmd/missing.rb

```diff
@@ -6,15 +6,41 @@ module Homebrew
   def missing
     return unless HOMEBREW_CELLAR.exist?
 
-    ff = if ARGV.named.empty?
+    formulae = if ARGV.named.empty?
       Formula.installed
     else
       ARGV.resolved_formulae
     end
+    if ARGV.json == "v1"
+      missing = print_missing_json(formulae)
+    else
+      missing = print_missing(formulae)
```

Comment:
Personally, I would approve of a separate commit (as part of the PR that changes `outdated`) that would clean up the style therein. I'm not sure other maintainers agree with me on that.

Either way, the commands are dealing with sufficiently different things that their code should mostly remain separated. If refactoring some common code into a shared utility function or something like that makes sense, that could be done. (But I currently don't see enough overlap between the two and where that would be the case.)